docs/howto/start-agent-remotely.md

```diff
@@ -0,0 +1,73 @@
+# How to start the agent remotely
+
+Ubuntu Pro for WSL, being a Microsoft Store application, cannot ship user services as of the time of this writing (late
+2024), but can deploy startup tasks instead, programs that run with user permissions when the said user logs on the
+Windows device. The UP4W background agent runs as a startup task. By design a startup task is only enabled by the
+operating system when the user interacts with the application for the first time. While this behaviour is a feature for
+end users it presents a source of friction for deployments at scale, when system administrators expect zero-touch
+deployment of UP4W to just work.
+
+This guide shows how system administrators can leverage Windows remote management solutions to start the agent once
+with user credentials, thus interacting with the application in user's behalf. Subsequent logons will have the UP4W
+background agent automatically as a consequence of such interaction.
+
+While this guide uses Intune, readers are expected to translate the steps into any remote management solution of their
+choice, as long as the said solution can run scripts with current user's credentials.
+
+## Pre-requisites
+
+- At least one managed Windows device.
+- A Windows remote management solution.
+
+## Key takeaways
```

Comment:
Again, consider Overview or Summary.

I think this section will be more useful when the initial explanatory material is removed at a later date.